<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_qsqlquery</class>
 <widget class="QDialog" name="dlg_qsqlquery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>qstrType</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbType">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>QMYSQL</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>qstrHostName</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbHostName">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>localhost</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>qstrDatabaseName</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbDatabaseName">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>hyhj</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>qstrUserName</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbUserName">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>root</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>qstrPassword</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbPassword">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>723</width>
            <height>597</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="1" colspan="3">
               <widget class="QLineEdit" name="leInsert">
                <property name="text">
                 <string>INSERT INTO t_forecast_user (id, name, info, info2, info3) VALUES (:id, :name, :info, :info2, :info3)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="3">
               <widget class="QLineEdit" name="leSelect">
                <property name="text">
                 <string>SELECT id, name, info, info2, info3 FROM t_forecast_user ORDER BY id</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>insert part sql</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="3">
               <widget class="QLineEdit" name="leDelete">
                <property name="text">
                 <string>DELETE FROM t_forecast_user WHERE id=:id AND name=:name AND info=:info AND info2=:info2 AND info3=:info3</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>delete sql</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="4">
               <widget class="QPushButton" name="pbSelect">
                <property name="text">
                 <string>执行</string>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>result</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0" colspan="4">
               <widget class="QPushButton" name="pbSelectIntoContainer">
                <property name="text">
                 <string>执行</string>
                </property>
               </widget>
              </item>
              <item row="9" column="2" colspan="2">
               <widget class="QListWidget" name="lwSelectResult"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>insert sql</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0" colspan="4">
               <widget class="QPushButton" name="pbInsert">
                <property name="text">
                 <string>执行</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>select sql</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" colspan="3">
               <widget class="QLineEdit" name="leInsertPart">
                <property name="text">
                 <string>INSERT INTO t_forecast_user (name, info, info2, info3) VALUES (:name, :info, :info2, :info3)</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0" colspan="4">
               <widget class="QPushButton" name="pbInsertPart">
                <property name="text">
                 <string>执行</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0" colspan="4">
               <widget class="QPushButton" name="pbDelete">
                <property name="text">
                 <string>执行</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>257</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
